packages/engine/Source/DataSources/SampledProperty.js

```diff
@@ -674,6 +674,30 @@ SampledProperty.prototype.addSamples = function (
   this._definitionChanged.raiseEvent(this);
 };
 
+/**
+ * Gets the time of the sample. Negative for backward.
```

Comment:
```suggestion
 * Retrieves the time of the provided sample associated with the index. A negative index accesses the list of samples in reverse order.
```

Just some minor tweaks for clarity.